f8a_worker/workers/graph_importer.py

```diff
@@ -35,7 +35,8 @@ def execute(self, arguments):
             {
                         'ecosystem': arguments['ecosystem'],
                         'name': arguments['name'],
-                        'version': arguments.get('version')
+                        'version': arguments.get('version'),
+                        'source_repo': arguments.get('source_repo', None)
```

Comment:
There will be no `source_repo` arg :) `arguments.get('ecosystem')` is what graph considers `source_repo`.